src/network/clientpackethandler.cpp

```diff
@@ -773,14 +773,19 @@ void Client::handleCommand_PlaySound(NetworkPacket* pkt)
 	v3f pos;
 	u16 object_id;
 	bool loop;
+	float fade = 0;
 
-	*pkt >> server_id >> name >> gain >> type >> pos >> object_id >> loop;
+	try	{
+		*pkt >> server_id >> name >> gain >> type >> pos >> object_id >> loop >> fade;
+	} catch (...) {
+		*pkt >> server_id >> name >> gain >> type >> pos >> object_id >> loop;
```

Comment:
this catch is absolutely wrong because you already read all values. Please let an empty catch.